__tests__/cli/add.js

```diff
@@ -0,0 +1,79 @@
+/* @flow */
+
+import NoopReporter from '../../src/reporters/base-reporter.js';
```

Comment:
Is it necessary to have e2e tests via exec calls?
We can't mock internals then.